docs/tutorials/developer-journey/level-5/5.2-ICP-ETH-tutorial.md

```diff
@@ -0,0 +1,433 @@
+# 5.2 Deploying an ETH starter project on the IC
```

Comment:
@jennifertrin We should add your link your tutorial to adapt the starter project for ERC-20 tokens.